lib/ansible/modules/cloud/docker/docker_volume.py

```diff
@@ -68,6 +68,12 @@
 requirements:
     - "python >= 2.6"
     - "docker-py >= 1.10.0"
```

Comment:
Per change log, this should be `1.5.0`, but align this with `MIN_DOCKER_VERSION = 1.7.0`, it should not be listed explictly here.

Reply:
Actually this is document, keep this and align to `1.7.0`.

Reply:
It uses the `labels` parameter for the `create_volume` call, so according to https://docker-py.readthedocs.io/en/stable/change-log.html it requires 1.10.0.